examples/storybook/src/stories/Hits.stories.ts

```diff
@@ -25,6 +25,19 @@ storiesOf('Hits', module)
       `,
     }),
   }))
+  .add('with escapeHTML', () => ({
+    component: wrapWithHits({
+      template: `
+      <ais-hits [escapeHTML]="false"> 
+        <ng-template let-hits="hits">
+          <div *ngFor="let hit of hits">
+            <span [innerHTML]="hit.name"></span>
+          </div>
+        </ng-template>
+      </ais-hits>
+      `,
+    }),
+  }))
```

Comment:
I've tested this works by creating an index with HTML strings inside but I couldn't find any HTML string in `instant_search` data. 
I don't see this being tested in Vanilla stories either. 

Any better ideas to make sure this keeps working ?

Reply:
We don't necessarily need to create stories for this type of options IMO. Here it might make it confusing for the person going through the story because nothing visually changes.

Reply:
story would only be useful with an XSS index, so either we can make one or remove the story

Reply:
I try to a certain extent to have a showcase if every option we have.
The rational behind it is that we can later run integration tests on them and be able to detect any changes. 

I guess for escapeHTML it's fine no to have it like @francoischalifour says. So i'll just remove the related commit from this PR.